scalaworker/src/mill/scalaworker/ScalaWorker.scala

```diff
@@ -200,7 +200,7 @@ class ScalaWorker(ctx0: mill.util.Ctx) extends mill.scalalib.ScalaWorkerApi{
             args: Seq[String])
            (implicit ctx: mill.util.Ctx.LogCtx): (String, Seq[Result]) = {
 
-    Jvm.inprocess(entireClasspath, classLoaderOverrideSbtTesting = true, cl => {
+    Jvm.inprocess(entireClasspath, Some(getClass.getClassLoader), cl => {
```

Comment:
We don't want to pass the whole classloader to the test run because the tests shouldn't need Zinc or Scala212 or other things on their classpath.